frontend/src/metabase/dashboard/components/DashboardGrid.jsx

```diff
@@ -248,44 +270,18 @@ export default class DashboardGrid extends Component {
     );
   }
 
-  renderMobile() {
-    const { width } = this.props;
-    const { dashcards } = this.state;
-    return (
-      <div
-        className={cx("DashboardGrid", {
-          "Dash--editing": this.isEditingLayout,
-          "Dash--dragging": this.state.isDragging,
-        })}
-      >
-        {dashcards &&
-          dashcards.map(dc => (
-            <div
-              key={dc.id}
-              className="DashCard"
-              style={{
-                width: width,
-                marginTop: 10,
-                marginBottom: 10,
-                height:
-                  // "text" cards should get a height based on their dc sizeY
-                  dc.card.display === "text"
-                    ? MOBILE_TEXT_CARD_ROW_HEIGHT * dc.sizeY
-                    : width / MOBILE_ASPECT_RATIO,
-              }}
-            >
-              {this.renderDashCard(dc, {
-                isMobile: true,
-                gridItemWidth: width,
-              })}
-            </div>
-          ))}
-      </div>
-    );
-  }
+  renderGridItem = ({ item: dc, breakpoint, gridItemWidth }) => (
```

Comment:
If this function returns a component, would it make sense to name it `GridItem` ?

Reply:
I guess for class component methods that render part of the UI, it's more common to use `renderSomething` names

Reply:
ah yeah. I forgot that.